API.md

```diff
@@ -7,6 +7,7 @@
 - `name` - determines the name of the cookie used to store session information. Defaults to _session_.
 - `maxCookieSize` - maximum cookie size before using server-side storage. Defaults to 1K. Set to zero to always use server-side storage.
 - `storeBlank` - determines whether to store empty session before they've been modified. Defaults to _true_.
+- `errorOnCacheNotReady` - will cause yar to throw exception if trying to persist to cache when cache is unavailable. Setting to false will allow application using yar to run uninterrupted if cache is not ready (however sessions will not be saving). Defaults to _false_.
```

Comment:
Tyler's original issue described all this.  The current behavior of yar throws when the cache isn't available.  This option allows yar to continue on without the cache, if desired, and I think that's a useful option.  The current behavior of yar is equivalent to `errorOnCacheNotReady = true`, which is why I want that to default to true.  If you choose to set it to false to avoid the thrown errors, then that's your choice.